templates/blog/post_detail.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>{{ post.title }}</title>
+{% endblock %}
+
+{% block content %}
+  <h2>{{ post.title }}</h2>
+  <div>
+    <div> Author: {{ post.owner }} Created: {{ post.created_time }}</div>
```

Comment:
It's a good practice to format the date and time to a more readable format using Django's date filter. For example, you could use `{{ post.created_time|date:'F j, Y' }}` to display the date in a 'Month Day, Year' format.